src/content/docs/en/guides/authentication.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Authentication
+description: An intro to authentication in Astro
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Authentication and authorization are two security processes that help you control who has access to different parts of your website or app. Authentication is the process of verifying a vistor's identity, and authorization is the process of allowing visitors access to protected routes and resources.
+
+For example, it is possible to set up *authorization* to view a private page on your site by requiring a password. However, if the page does not contain any personal or sensitive information (e.g. a static page with additional content available only to those who know the password), you might not include any *authentication*.
+
+Authentication provides the most security for you and your visitors, and allows you to customize areas of your site for logged-in individuals.
+
+Use a trusted authentication library (e.g. [Lucia Auth](https://lucia-auth.com/), [Auth.js](https://authjs.dev/)) whenever sensitive information is involved. Authentication libraries provide utilities for multiple authentication methods, such as email signin and OAuth providers.
+
+:::tip
+You can find [community "auth" integrations](https://astro.build/integrations/?search=auth) in the integrations directory.
+:::
+
+## Lucia
+
+Lucia is a framework agnostic, session based authentication library with great Astro support.
+
+### Installation
+
+You can install Lucia using the package manager of your choice.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install lucia
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add lucia
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add lucia
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Configuration
+
+Use [Lucia's "Getting started in Astro"](https://lucia-auth.com/getting-started/astro) guide to initialize Lucia with an adapter and set up a database to store users and sessions.
+
+### Usage
+
+:::tip
+Follow one of Lucia's complete Astro tutorials to add [username and password authentication](https://lucia-auth.com/tutorials/username-and-password/astro) or [GitHub OAuth](https://lucia-auth.com/tutorials/username-and-password/astro) to your Astro project.
+:::
```

Comment:
```suggestion
Follow one of Lucia's complete Astro tutorials to add [username and password authentication](https://lucia-auth.com/tutorials/username-and-password/astro) or [GitHub OAuth](https://lucia-auth.com/tutorials/username-and-password/astro) to your Astro project.
```

This may not need to be an aside, as it directly relates to the process

Reply:
Admittedly, we're working on the best way to handle this, since Lucia has great guides and there isn't a lot of need for us to reproduce here. So, is it part of the process to "go do their tutorial?"  😅

Reply:
> Admittedly, we're working on the best way to handle this, since Lucia has great guides and there isn't a lot of need for us to reproduce here. So, is it part of the process to "go do their tutorial?" 😅

Yeah, I think so! Don't have to fix whats not broken, as well